### DISCUSSION TITLE
Variable_output vs. Material_output

### DISCUSSION THREAD

**Discussion by Joseph-0123** [idaholab/moose#23452] [Q&A General] [answered]
Dear all,

I get an unstable variable result as shown below. In addition, I also define a new property in the material which is equal to this variable. 

However, when I plot them together, I find they have quite different behavior. But essentially they should be the same. What is the reason for this? Dear @GiudGiud, do you have any idea?

J

![image](https://user-images.githubusercontent.com/56687392/219517701-cc3e6aef-f4c2-453c-b7db-2edcd9ef6e90.png)

**Comment by GiudGiud** [comment, marked as answer]
Conversion from variable to material involves a projection. I think most likely a const monomial though you should double check

This will smooth things out at output

**Reply by GiudGiud** [reply]
either that or something is off for variable output but it's well tested at this point

**Reply by Joseph-0123** [reply]
Thanks for your help.
